src/lib/eliom_common.server.mli

```diff
@@ -711,7 +711,7 @@ type eliom_js_page_data = {
 
 val get_site_dir : sitedata -> Url.path
 val get_site_dir_string : sitedata -> string
-val get_secure : bool option -> sitedata -> bool
+val get_secure : secure_o:bool option -> sitedata:sitedata -> unit -> bool
```

Comment:
To be honest, I don't think that `_o` improves clarity, because we can already see that it is an option from the type. Omitting it would just make passing around `secure` more concise.